load_testing/common_flows/flow_sp_ial2_sign_up.py

```diff
@@ -103,7 +103,7 @@ def ial2_sign_up(context):
         context,
         "post",
         "/sign_up/create_password",
-        "/two_factor_options",
+        "/authentication_methods_setup",
```

Comment:
cc @18F/identity-katherine just to double check, this URL change is permanent, not part of a feature flag rollout, right?

Reply:
correct, we wanted a clearer url to indicate what is actually happening.

Reply:
It sounds like the url change is permanent so we should be good to go, is that right?